src/test/java/net/minecraftforge/debug/DynBucketTest.java

```diff
@@ -180,6 +162,29 @@ public void onBucketFill(FillBucketEvent event)
         }
     }
 
+    @Mod.EventBusSubscriber(value = Side.CLIENT, modid = MODID)
+    public static class ClientEventHandler
```

Comment:
It may look cleaner, but its less hacky to use a separate class as side only effectively removes code at runtime.